src/puppeteer/puppeteer-controller.ts

```diff
@@ -5,9 +5,6 @@ import { noop } from '../utils';
 
 const PROCESS_KILL_TIMEOUT_MILLIS = 5000;
 
-/**
```

Comment:
Good question. I think they were there because of JSDoc build, but they might be obslete now. We should test whether docs build correctly without them.

Reply:
The docs build fine without them.